middle_end/flambda/types/env/typing_env.rec.ml

```diff
@@ -436,16 +438,19 @@ let print_with_cache ~cache ppf
       Scope.Map.filter (fun _ level -> not (One_level.is_empty level))
         levels
     in
+    let aliases = Cached.aliases (One_level.just_after_level current_level) in
```

Comment:
These changes to the printers should be in a separate PR please and guarded by a command-line flag.

Reply:
I don't agree with the command-line flag part, particularly when so much information is printed already.
But I'll push this change into a separate PR, so that it doesn't block this one.